config/hsc/processCcd.py

```diff
@@ -19,3 +20,6 @@
 
 # Do not use NO_DATA pixels for fringe subtraction.
 config.isr.fringe.stats.badMaskPlanes=['SAT', 'NO_DATA']
+
+config.charImage.detectAndMeasure.measurement.algorithms["base_Jacobian"].pixelScale = 0.168
+config.calibrate.detectAndMeasure.measurement.algorithms["base_Jacobian"].pixelScale = 0.168
```

Comment:
This is not the proper way to configure measurement plugins, see the Jacobian unit test in meas_base